src/functions-and-arrays.js

```diff
@@ -1,34 +1,71 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(numb1, numb2) {
+  if (numb1 < numb2) {
+    return numb2;
+} else if (numb1 > numb2) {
+    return numb1;
+  
+  } 
+}
 
```

Comment:
Hi Alex! I won't go into all the failed tests so much now, you can look at the solutions in the Student Portal again to see exactly what was asked for. But still a few notes:
for example in this iteration the test was asking for returning any number if the numbers are equal. You could have solved it like this:
```
function maxOfTwoNumbers(numb1, numb2) {
  if (numb1 < numb2) {
    return numb2;
  } else if (numb1 > numb2) {
    return numb1;
  }
  else {
    return numb1
  }
}

```